gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -787,7 +787,7 @@ public TraceRequest getTraceRequest(Settings settings) {
           options.addAllEnvironment(splitEnv(envVars.getText()));
         }
         if (config.getMidExecutionCaptureSupport() != Service.FeatureStatus.NotSupported) {
-          options.setDeferStart(settings.traceStartAt != 0);
+          options.setDeferStart(settings.traceStartAt < 0);
```

Comment:
Yikes. We probably should also fix this in the server... starting a trace at frame x is by definition a deferred start.

Reply:
Yes, it's a bit misleading. I think deferred start refers to "Start when the users clicks button".